ruby-sdk/ext/eppo_client/Cargo.toml

```diff
@@ -12,7 +12,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 env_logger = { version = "0.11.3", features = ["unstable-kv"] }
-eppo_core = { version = "=5.0.0", features = ["vendored"] }
+eppo_core = { version = "=4.1.1", features = ["vendored"] }
```

Comment:
If we're building from local source, it must depend on the latest (possibly unpublished) core. Otherwise, it's undefined what you're building — it's not the latest version and it's the previous version.

Reply:
@rasendubi I agree. However, I had to change this because the build was failing since it couldn't find `eppo_core` v5.0.0 - possibly it was trying to download it instead of using the local one.
How do I force it to build it from local instead?

Reply:
thank you!